azkaban-common/src/main/java/azkaban/executor/ExecutorApiClient.java

```diff
@@ -46,8 +123,9 @@ protected String parseResponse(final HttpResponse response)
 
     if (statusLine.getStatusCode() >= 300) {
```

Comment:
nit: This line is from old code but would it be possible to move the constant to a static variable?

Reply:
The comparison `>= 300` is for http error codes for non-success.  
I think in this instance converting it to a static variable will make the usage less clear for someone looking at this in future, esp as it will retain the comparison `>=`. 
A better alternative would be to replace with a method like `boolean isHttpReturnCodeSuccess(int code)` which encapsulates the comparison but that should be in a separate PR.